lib/src/clojure_lsp/crawler.clj

```diff
@@ -145,7 +148,7 @@
    client-settings
```

Comment:
probably this is the most critical function to change as it needs to update and get updated values all the time

Reply:
Yeah, elsewhere I tried to do `(let [db @db*] ,,,)` but not for this function. Clearly, it needs to do several sequential swaps, and so I didn't want to end up with a stale db anywhere.